lib/ex_gram/plug.ex

```diff
@@ -83,6 +83,11 @@ if Code.ensure_loaded?(Plug) do
       end
     end
 
+    defp webhook_path_match?(conn) do
+      webhook_path = ExGram.Config.get(:ex_gram, :webhook)[:webhook_path] || "telegram"
+      length(conn.path_info) == 2 && Enum.at(conn.path_info, 0) == webhook_path
+    end
```

Comment:
Now that we will suppor custom paths, it should support paths like '/my/bot/webhook/<token>`, so let's try to match all the path, removing the last part (that bust be a token)

```suggestion
    defp webhook_path_match?(conn) do
      webhook_path = ExGram.Config.get(:ex_gram, :webhook)[:webhook_path] || "telegram"
      webhook_path = String.trim_leading(webhook_path, "/")
      
      case Enum.drop(conn.path_info, -1) do
          [] -> false
          received_path ->  webhook_path == Path.join(received_path)
      end
    end
```